.github/workflows/release.yml

```
@@ -0,0 +1,80 @@
+name: Release
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    name: Release
+    permissions:
+      actions: write
+      checks: write
+      contents: write
+      deployments: write
+      discussions: write
+      id-token: write
+      issues: write
+      packages: write
+      pages: write
+      pull-requests: write
+      repository-projects: write
+      security-events: write
+      statuses: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set Env Variables
+        run: |
+          current_branch=${GITHUB_REF#refs/heads/}
```

I think we might need to change this to incorporate tags, this is what [Github's docs](https://docs.github.com/en/actions/learn-github-actions/variables) say. Specific quote here:

>This is only set if a branch or tag is available for the event type. The ref given is fully-formed, meaning that for branches the format is refs/heads/<branch_name>, for pull requests it is refs/pull/<pr_number>/merge, and for tags it is refs/tags/<tag_name>. For example, refs/heads/feature-branch-1

We typically trigger a push to production only when a tag is created, so would this be more correct then?
```suggestion
          current_branch=${GITHUB_REF#refs/tags/}
```